packages/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.ts

```diff
@@ -69,6 +69,8 @@ export const getAudioReceiverMqa = ({
     (totalFecPacketsDiscarded - lastFecPacketsDiscarded);
   audioReceiver.common.fecPackets = fecRecovered;
 
+  audioReceiver.common.rtpRecovered = fecRecovered;
```

Comment:
This is just the same value we assign to a different variable. Is this correct? If so, what is the problem in using the existing one `fecPackets`?

Reply:
As it stands, FEC is the only method for audio packet recovery from the web client. This means that for audio receivers, the `fecPackets` count will directly reflect the `rtpRecovered` stats.

Reply:
I see. Got it.